### ISSUE TITLE
Change the status of the issue

### ISSUE THREAD

**Issue by suchandd** [rcarz/jira-client#150]
May i know how to change the status of the issue like change the status of issue from open to close using your client please??

**Comment by nach-o-man** [comment]
@suchandd You can't just change the status via standard API(which is used in this client) - you have to make all necessary workflow transitions of the issue. Like this:
```java
/* Now let's start progress on this issue. */
issue.transition().execute("Start Progress");
// an so on ...
```

**Comment by suchandd** [comment]
Thanks for the early reply,i fixed it using 
issue.transition().execute("Closed");
is it possible to completely delete the issue (i think not possible)??

**Comment by nach-o-man** [comment]
Not yet, but I have pull request pending for that  #139.

**Comment by suchandd** [comment]
is it done??please let me know when it is done thanks a lot , how do you close multiple issues like issues having the same name??

**Comment by rcarz** [comment]
Thanks! Rebased and merged in ba58b45df6284523de416405cc7a5b0e1ee19ae1
